uniflow/pipeline.py

```diff
@@ -4,29 +4,29 @@
 from threading import Thread
 from typing import Any, List, Mapping
 
-from uniflow.flow.config import PipelineConfig
 from uniflow.flow.client import ExtractClient, TransformClient
+from uniflow.flow.config import PipelineConfig
 from uniflow.op.prompt_schema import Context
 
 
-class MultiProcessPipeline:
+class MultiThreadPool:
```

Comment:
Per our discussion, this is a multi stage pipeline, at each stage such as extract, or transform, there is a ThreadPool. However, this does not mean you can just call it MultiThreadPool because in Python, [ThreadPool](https://docs.python.org/3/library/threading.html) has its own meaning and this is very confusing. You should name it `MultiFlowsPipeline`.